components/friend.php

```diff
@@ -0,0 +1,337 @@
+<?php
+/**
+ * Manage BuddyPress Friends.
+ *
+ * @since 1.6.0
+ */
+class BPCLI_Friend extends BPCLI_Component {
+
+	/**
+	 * Object fields.
+	 *
+	 * @var array
+	 */
+	protected $obj_fields = array(
+		'id',
+		'initiator_user_id',
+		'friend_user_id',
+		'is_confirmed',
+		'is_limited',
+	);
+
+	/**
+	 * Create a new friendship.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <initiator>
+	 * : ID of the user who is sending the friendship request. Accepts either a user_login or a numeric ID.
+	 *
+	 * <friend>
+	 * : ID of the user whose friendship is being requested. Accepts either a user_login or a numeric ID.
+	 *
+	 * [--force-accept=<force-accept>]
```

Comment:
Can `--force-accept` and `--silent` be flags instead? Like `--porcelain`.

Reply:
`--silent` convention is used all over without being a flag. If we are going to change here, I'd recommend updating in the other components as well.

Reply:
Used all over wp-cli-buddypress or all over the wp-cli world? If it's just us, I'd prefer to make it a flag throughout. `--foo=false` is not idiomatic for the command line and should be avoided.

Reply:
I meant all over in wp-cli-buddypress, I'm not sure how it is used on wp-cli. Will update it! :)